src/current/_includes/cockroachcloud/backups/terraform-managed-backups.md

```diff
@@ -0,0 +1,9 @@
+Set the following for the `backup_config` attribute:
+
+- `enabled` controls whether managed backups are enabled or disabled. If you are disabling managed backups, you cannot set backup frequency or retention. Possible values are: `true` or `false`.
+- `frequency_minutes` determines [how often](#frequency) the managed backup will run in minutes. Possible values are: `5`, `10`, `15`, `30`, `60`, `240`, `1440`.
+- `retention_days` sets the number of days Cockroach Labs will [retain](#retention) the managed backup in storage. You can change `retention_days` for the cluster **once** (whether in Terraform, the [Cloud API](#cloud-api), or the [Cloud Console](#cloud-console)). Possible values are: `2`, `7`, `30`, `90`, `365`. Note that:
+    - If the initial value of the `retention_days` attribute is the default value `30`, you'll be able to modify the backup retention setting once more. 
+    - If the initial value is not the default, you will not be able to modify `retention_days` again. You can refrain from including `retention_days` in the Terraform configuration and instead manage the retention in the Cloud Console.
+    
+    For more details on modifying `retention_days` more than once, refer to the [Updating Backup Retention](https://github.com/cockroachdb/terraform-provider-cockroach/blob/main/docs/guides/updating-backup-retention.md) documentation in the Terraform provider for CockroachDB {{ site.data.products.cloud }} Repository.
```

Comment:
should this just be to reach out to support since we dont let you do it on cloud api anyway?

Reply:
I kept this sentence so users can find the terraform provider docs, and also added the link to contact support for modifying the setting again. Hope that's good.